pkg/skuba/actions/node/bootstrap/bootstrap.go

```diff
@@ -51,6 +51,8 @@ func Bootstrap(bootstrapConfiguration deployments.BootstrapConfiguration, target
 			fmt.Printf("[bootstrap] node %q has already the core components bootstrapped\n", target.Target)
 			coreBootstrapDone = true
 		}
+	} else {
+		return err
```

Comment:
This needs to be removed, otherwise bootstrap won't work (when bootstrapping we have no `admin.conf` yet).

Reply:
Then we should remove below code:
```golang
if clientSet, err := kubernetes.GetAdminClientSet(); err == nil {
		_, err := clientSet.ServerVersion()
		if err == nil {
			fmt.Printf("[bootstrap] node %q has already the core components bootstrapped\n", target.Target)
			coreBootstrapDone = true
		}
	} 
```
I am trying to install CaaSP 4.0 on my VMs, and I start with
```shell
skuba cluster init --control-plane 192.168.100.246 caasp4.0
cd caasp4.0
skuba node bootstrap --user hui-zhi --sudo --target 192.168.100.246 master00
```
But it failed and shows:
```
[bootstrap] updating init configuration with target information
[bootstrap] writing init configuration for node
[bootstrap] applying init configuration to node
F0913 21:23:47.759498   12999 bootstrap.go:49] error bootstraping node: failed to apply state kubeadm.init: Process exited with status 1
```
I though it caused by I don't have ```admin.conf```, because the code mislead me. Do you have any suggestion how can I resolve it?

Reply:
The `bootstrap` action can be called multiple times. If a bootstrap action worked on the `kubeadm` operation, but failed afterwards when creating addons, the rerun of bootstrap will detect that it already has a working node by fetching the server version using the admin client set, and go on to only redeploy addons. This can be improved so instead of checking the server version we can do a somewhat more complete test, but is the reasoning behind this.

That's why this operation should not fail right away, in fact, every first `bootstrap` call will fail to fetch the admin clientset at that point in time, and it's fine.

Reply:
As of how to resolve, just remove this `else` branch :)